temporal-sdk/src/main/java/io/temporal/testing/TestEnvironmentOptions.java

```diff
@@ -114,6 +143,14 @@ public Scope getMetricsScope() {
     return metricsScope;
   }
 
+  public boolean isUseExternalService() {
```

Comment:
Docstring for what this means would be useful as well

Reply:
Done